src/bindings.rs

```diff
@@ -153,6 +153,7 @@ impl Binding {
                     | ScrollRightColumns(_)
                     | ScrollLeftScreenFraction(_)
                     | ScrollRightScreenFraction(_)
+                    | AppendDigitToRepeatCount(_)
```

Comment:
Putting this here makes it fill up the help screen for navigation.  Probably we want to hide these, so we should make a new `Category::Internal` or `Category::Hidden` and map these actions to that.  The help screen should skip that category.
<img width="503" alt="Screenshot 2021-06-28 at 09 34 46" src="https://user-images.githubusercontent.com/5471678/123605823-28b6e900-d7f4-11eb-9985-f5864994e926.png">